xml/System.Windows.Forms/Control.xml

```diff
@@ -4284,7 +4284,7 @@ The following table lists Windows Forms controls and which event (<xref:System.W
         <ReturnType>System.Windows.Forms.Padding</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Gets the internal spacing, in pixels, of the contents of a control.</summary>
+        <summary>Gets the default internal spacing, in pixels, of the contents of a control.</summary>
         <value>A <see cref="T:System.Windows.Forms.Padding" /> that represents the internal spacing of the contents of a control.</value>
```

Comment:
```suggestion
        <value>The default internal spacing of the contents of a control.</value>
```

Reply:
Oh, of course, you're right, I forget :p